pkg/api/persistentvolume/util_test.go

```diff
@@ -30,10 +30,12 @@ import (
 func TestPVSecrets(t *testing.T) {
 	// Stub containing all possible secret references in a PV.
 	// The names of the referenced secrets match struct paths detected by reflection.
+	secretNamespace := "Spec.PersistentVolumeSource.AzureFile.SecretNamespace"
 	pvs := []*api.PersistentVolume{
 		{Spec: api.PersistentVolumeSpec{PersistentVolumeSource: api.PersistentVolumeSource{
-			AzureFile: &api.AzureFileVolumeSource{
```

Comment:
instead of replacing this, add a second PV with a namespace, so we exercise both cases